docs/debugging.md

```diff
@@ -202,6 +196,25 @@ Alternatively, select "Settings" from the Dev Menu, then update the "Debug serve
 
 > If you run into any issues, it may be possible that one of your Chrome extensions is interacting in unexpected ways with the debugger. Try disabling all of your extensions and re-enabling them one-by-one until you find the problematic extension.
 
+## Remote debugging
+
+Starting from version 0.72, Remote debugging has been deprecated in favor of other alternatives such as Direct debugging with Safari. However, if your project still relies on this feature, you can manually enable it manually through the `NativeDevSettings.setIsDebuggingRemotely` function.
```

Comment:
- link to the direct debugging with safari section in that text
- we should include some context on why it was deprecated
- we should indicate that this will be removed entirely in 73 but until then the workaround will be available

Reply:
Updated